src/styles/blocks.scss/card.scss

```diff
@@ -0,0 +1,105 @@
+.card {
+  font-family: Roboto, sans-serif;
```

Comment:
It's better to define the font family in a general stylesheet file, not in a block file. This way, you can ensure consistency across your entire project.